03_Uitvoering.Rmd

```diff
@@ -2668,16 +2561,158 @@ Elke gridcel waar het plaatsen van een pb gewenst is, wordt eerst gecategoriseer
+             gebied = area_name,
+             watinacode =  loc_code
+             ) %>%
+      arrange(rastercelnummer, GTgroep),
+      caption =  "Overzicht van de geselecteerde peilbuizen"
+      )
+if (pdfmaken == TRUE) {
+  kobj
+} else {
+  kobj %>%
+    kable_styling(bootstrap_options = c("striped", "hover", "condensed", "responsive"),
+                full_width = T,
+                position = "left",
+                font_size = 8,
+                fixed_thead = T
+                ) %>%
+   # column_spec(1:3, bold = F, border_right = F, width = "35em") %>%
+   # column_spec(c(3,5:7, 9:10), width = "1cm") %>%
+   # row_spec(0, angle = -90, align = "c")  %>% #fixeer veldnamen
+    scroll_box(height = "250px")
+}
+
+```
+
+<br/><br/>
+
+```{r synthese-reserve-tubes}
+
+kobj <- kable(tubes_reserve %>%
+        dplyr::select(rasterid, groupnr, area_name, loc_code, x, y) %>%
+        rename(GTgroep = groupnr,
+               rastercelnummer = rasterid,
+               gebied = area_name,
+               watinacode =  loc_code
+               ) %>%
+        arrange(rastercelnummer, GTgroep),
+      caption =  "Overzicht van de reserve peilbuizen"
+      )
+if (pdfmaken == TRUE) {
+  kobj
+} else {
+  kobj %>%
+    kable_styling(bootstrap_options = c("striped", "hover", "condensed", "responsive"),
+                full_width = T,
+                position = "left",
+                font_size = 8,
+                fixed_thead = T
+                ) %>%
+   # column_spec(1:3, bold = F, border_right = F, width = "35em") %>%
+   # column_spec(c(3,5:7, 9:10), width = "1cm") %>%
+   # row_spec(0, angle = -90, align = "c")  %>% #fixeer veldnamen
+    scroll_box(height = "250px")
+}
+
+```
+
+<br/><br/>
+
+Figuur \@ref(fig:tubes-selected-plot) toont de geselecteerde peilbuizen op kaart. 
+Er wordt een onderscheid gemaakt tussen de meetpunten die meetpunten die reeds inzetbaar zijn en de meetpunten waarbij er nog extra metingen nodig zijn.
+
+```{r tubes-selected-plot, fig.cap = "de ligging van de geselecteerde peilbuizen"}
+tubes_selected_sf <- as_points(tubes_selected)
+
+tubes_selected_tm <- raster_meetnet_poly_tm + 
+  tm_shape(tubes_selected_sf %>% mutate(Inzetbaarheid = factor(if_else(selectie == -1, "potentieel", "actueel")))) + 
+  tm_symbols(size = 0.25, shapes.labels = "loc_code", col = "Inzetbaarheid", clustering = FALSE) + tm_layout(title = "geselecteerde peilbuizen", asp = 0 )
+
+tubes_selected_tm
+```
+
+
+```{r tubes-as-shape, warning=FALSE}
+
+write_sf(tubes_selected_sf %>%  
+           select(-obswell_statecode, -loc_validitycode) %>% 
+           rename(watinac = "loc_code"), 
+         driver = "ESRI Shapefile", file.path("data","GIS", "tubes_selected_2020_05.shp"))
+
+write_sf(tubes_reserve %>% 
+           as_points() %>%  
+           select(-obswell_statecode, -loc_validitycode) %>% 
+           rename(watinac = "loc_code"), 
+         file.path("data","GIS", "tubes_reserve_2020_05.shp" ))
```

Comment:
It's not bad to use shapefiles, but it's better to use the geopackage format. Single-file, open, better capabilities.

Reply:
I added the files in the geopackage-format.

Reply:
Superb.